docs/Formula-Cookbook.md

```diff
@@ -748,37 +748,23 @@ For `url`/`regex` guidelines and additional `livecheck` block examples, refer to
 
 ### Excluding formula from autobumping
 
-By default, all new formulae in the Homebrew/core repository are autobumped. It means that future updates will be handled automatically by Homebrew CI jobs, and contributors do not have to do it manually.
+By default, all new formulae in the `Homebrew/homebrew-core` repository are autobumped. This means that future updates are handled automatically by Homebrew CI jobs, and contributors do not have to submit pull requests.
```

Comment:
```suggestion
By default, all new formulae in the Homebrew/homebrew-core repository are autobumped. This means that future updates are handled automatically by Homebrew CI jobs, and contributors do not have to submit pull requests.
```
No need to make this monospace. This is inconsistent with other uses even within this PR.

Reply:
@AlternateRT Ok, thanks. Happy either way then but, if you'd be willing, making that consistent across our docs would be a great future PR. If you're not interested: shout and I'm sure @EricFromCanada would be 😁

Reply:
On my list is the idea to distinguish between mentions of a _tap_ as "`homebrew/core`" vs. a _repository_ as "Homebrew/homebrew-core". (It's a long list.)